src/data-table/index.js

```diff
@@ -59,10 +58,13 @@ class DataTable extends Component {
       defaultSortDir: sortDirection,
     } = props
 
+    const sortCol = this.columns().find(col => col.dataKey === sortColumn)
+
     const picked = this.pickables()
     this.state = {
       sortColumn,
       sortDirection,
+      sortType: sortCol ? sortCol.sortType : false,
```

Comment:
why keep the `sortType` state here if we already ensured that every column has a `sortType`, we can get the `sortType` by `sortColumn`